ibis/tests/expr/test_table.py

```diff
@@ -123,6 +123,19 @@ def test_table_tab_completion():
     assert items.issuperset(table.columns)
 
 
+def test_keys(table):
+    keys = table.keys()
+    assert keys == table.columns
+
+
```

Comment:
```suggestion
```

The test below seems sufficient. Alternatively you can put this assertion there.